src/mlpack/bindings/java/maven/CLI.java

```diff
@@ -0,0 +1,536 @@
+package org.mlpack;
+
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.buffer.DataBuffer;
+import org.nd4j.linalg.api.buffer.DataType;
+import org.nd4j.linalg.factory.Nd4j;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bytedeco.javacpp.*;
+import org.bytedeco.javacpp.annotation.*;
+
+@Platform(include = {"cli_util.hpp", "deleter.hpp"})
+@Namespace("mlpack::util")
+class CLI {
```

Comment:
We can do that. It's only a matter of preference. It would look somewhat weird though. Afaik, every code style guide for Java requires brackets to be on the same line.